### DISCUSSION TITLE
09.aug.2024

### DISCUSSION THREAD

**Discussion by S1lverone** [Lailloken/Exile-UI#454] [Q&A]
hey guys the sscript ahs stop working with item info and map info as of today. it worked yesterday. any reason for this?
cheers

**Comment by Lailloken** [comment]
Item-related omni-key features now require the inventory pixel-check to be set up.

**Comment by S1lverone** [comment]
I dissbled both. And today I had to close the entire script. Even though i was only running with the clone feature, poe's "bad" software coding could not handle me running both.
But it is only when i have the pixel background enabled that the mouse freezes or lags. Anything i can do to test or help out?

Sendt fra Outlook til Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Lailloken** [reply]
> Anything i can do to test or help out?

Not really. I'll release a hotfix shortly to make item-related features accessible again with disabled background checks. Actually, these stutters are happening because of some sort of incompatibility, so I guess you could try to find out if it's due to specific software that's also running on your PC. As I hinted earlier, you're only the second person (that I know of) who has to use this workaround to avoid stutters.

**Comment by S1lverone** [comment]
Ok Impressive. Mean while I will look into buying a pc that specific can handle Path of Exiles huge CPU usage Spikes. I will make a purchase during this week, it should be ready and deliverede for me to test your adjustments with the new hotfix.
Looking forward to grow an .ahk community🙌

Sendt fra Outlook til Android<https://aka.ms/AAb9ysg>
________________________________
